src/System.Private.Uri/tests/FunctionalTests/IriTest.cs

```diff
@@ -64,32 +64,35 @@ public void Iri_Uri_ShouldNotThrowArgumentOutOfRange()
         public void Iri_Uri_SchemaParsing_ShouldNotThrowArgumentOutOfRange()
         {
             string root = "viewcode://./codeschema_class?";
-            string uriDataFra = root + Uri.EscapeDataString("Type=\u00E9");
+            string uriDataFra = root + "Type=" + Uri.EscapeDataString("\u00E9");
 
             Uri u1 = new Uri(uriDataFra);
 
-            // TODO #8330 : Normalization should produce the same result for escaped/unescaped URIs.
-            // Assert.Equal(root + "Type=%C3%A9", u1.AbsoluteUri);
-
-            Assert.NotEqual(root + "Type=%C3%A9", u1.AbsoluteUri);
+            Assert.Equal(root + "Type=%C3%A9", u1.AbsoluteUri);
         }
 
         [Fact]
-        public void Iri_IncorrectNormalization()
+        public void Iri_ReservedCharacters_NotNormalized()
         {
             Uri u1 = new Uri(@"http://test.com/%2c");
             Uri u2 = new Uri(@"http://test.com/,");
 
-            // TODO #8330 : Normalization should produce the same result for escaped/unescaped URIs.
-            //Assert.Equal(
-            //    u1.ToString(),
-            //    u2.ToString());
-
             Assert.NotEqual(
                 u1.ToString(),
                 u2.ToString());
         }
 
+        [Fact]
+        public void Iri_UnknownSchemeWithoutAuthority_DoesNormalize()
```

Comment:
Will the new test pass on Framework?

Reply:
>Will the new test pass on Framework?

I agree this test probably needs a SkipOnTargetFramework etc.

Reply:
Yep, you're right. I'll add that now.